python/shotgun_entities/card_delegate.py

```diff
@@ -0,0 +1,249 @@
+# Copyright (c) 2016 Shotgun Software Inc.
+# 
+# CONFIDENTIAL AND PROPRIETARY
+# 
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your 
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import sgtk
+from sgtk.platform.qt import QtCore, QtGui
+from .card_widget import ShotgunEntityCardWidget
+
+views = sgtk.platform.current_bundle().import_module("views")
+shotgun_model = sgtk.platform.import_framework(
+    "tk-framework-shotgunutils",
+    "shotgun_model",
+)
+
+class ShotgunEntityCardDelegate(views.EditSelectedWidgetDelegate):
+    """
+    A delegate wrapper for :class:`~ShotgunEntityCardWidget`.
+
+    :ivar show_labels:              Whether to show labels for any Shotgun
```

Comment:
so what's the story here? maybe we can discuss a bit? We used to have widgets as part of the qtwidgets fw, but then we pulled them out because we felt it was premature.